db/view/view.cc

```diff
@@ -491,7 +491,7 @@ mutation_partition& view_updates::partition_for(partition_key&& key) {
 }
 
 size_t view_updates::op_count() const {
-    return _op_count++;;
+    return _op_count;
```

Comment:
We can probably de-mutable view_updates::_op_count now.

Generally, adding mutable means "I'm hiding a bug here".

Reply:
Ah you did that.